src/mainwindow.cpp

```diff
@@ -286,9 +286,9 @@ void MainWindow::closeEvent(QCloseEvent* event)
 
 QString MainWindow::queryOpenDataFile()
 {
-    const auto filter = tr("Hotspot data Files (perf*.data perf.data.* *.perfparser);;"
+    const auto filter = tr("Hotspot data Files (perf*.data perf.data.* *.perfparser *.perfparser.*);;"
                            "Linux Perf Files (perf*.data perf.data.*);;"
-                           "Perfparser Files (*.perfparser);;"
+                           "Perfparser Files (*.perfparser *.perfparser.*);;"
```

Comment:
That was my thought, too; but where can I find the list of supported extensions (and is it always an "all extensions or KFArchive not set")?

Reply:
ah, I realize that there is sadly no API in `KCompressionDevice` to query the supported extensions. I guess it's fine to leave it it as-is then, but since KArchive is optional, make it at least so that we don't allow loading `*.perfparser.*` in such cases (and stick to the singular pattern like before?)

Reply:
We already allow `perf.data.*` no matter if compression support is enabled or not (= including perf.data.prog1, perf.data.prog2), ´showing that "perf.data.gz" is just a side.case; I tend to say "we only allow the same for perfparser now".
What do you think?

Reply:
true, let's leave it as-is then